lib/hammer_cli_foreman/table_preferences.rb

```diff
@@ -0,0 +1,71 @@
+module HammerCLIForeman
+  class TablePreferences < HammerCLIForeman::Command
+    resource :table_preferences, :users
+    command_name "table-preferences"
+
+    class ListCommand < HammerCLIForeman::ListCommand
+      output do
+        field :id, _("Id")
+        field :name, _("Name")
+        field :columns, _("Columns")
+        field :created_at, _("Created at"), Fields::Date
+      end
+
+      build_options
+    end
+
+    class CreateTablePreferences < HammerCLIForeman::CreateCommand
+      command_name 'create'
+
+      def request_params
+        params = super
+        params['name'] = params['table_preference']['name']
+        params['columns'] = params['table_preference']['columns']
+        params
+      end
+      
+      success_message _('Created table preferences.')
+      failure_message _('Could not create table preferences')
+
+      build_options
+    end
+
+    class UpdateTablePreferences < HammerCLIForeman::UpdateCommand
+      command_name 'update'
+
+      def request_params
+        params = super
+        params['name'] = params['table_preference']['name']
+        params['columns'] = params['table_preference']['columns']
+        params
+      end
+
+      success_message _('Table preferences was updated.')
+      failure_message _('Could not update table preferences')
+
+      build_options
+    end
+
+    class InfoTablePreferences < HammerCLIForeman::InfoCommand
+      output do
+        field :id, _("Id")
+        field :name, _("Name")
+        field :columns, _("Columns")
+        field :created_at, _("Created at"), Fields::Date
+        field :updated_at, _("Updated at"), Fields::Date
+      end
```

Comment:
We can reduce duplication by using:
```suggestion
      output ListCommand.output_definition do
        field :created_at, _("Created at"), Fields::Date
        field :updated_at, _("Updated at"), Fields::Date
      end
```